discovery-provider/src/trending_strategies/aSPET_trending_tracks_strategy.py

```diff
@@ -114,8 +114,8 @@ def update_track_score_query(self, session):
                         WHEN tp.owner_follower_count < :y
                             THEN 0
                         WHEN EXTRACT(DAYS from now() - aip.created_at) > :year
-                            THEN greatest(1.0/:q, pow(:q, greatest(-10, 1.0 - 1.0*EXTRACT(DAYS from now() - aip.created_at)/:year))) * (:N * aip.year_listen_counts + :F * tp.repost_year_count + :O * tp.save_year_count + :R * tp.repost_count + :i * tp.save_count) * tp.karma
-                        ELSE (:N * aip.year_listen_counts + :F * tp.repost_year_count + :O * tp.save_year_count + :R * tp.repost_count + :i * tp.save_count) * tp.karma
+                            THEN greatest(1.0/:q, pow(:q, greatest(-10, 1.0 - 1.0*EXTRACT(DAYS from now() - aip.created_at)/:year))) * (:F * tp.repost_year_count + :O * tp.save_year_count + :R * tp.repost_count + :i * tp.save_count) * tp.karma
+                        ELSE (:F * tp.repost_year_count + :O * tp.save_year_count + :R * tp.repost_count + :i * tp.save_count) * tp.karma
```

Comment:
We no longer use this trending right, so this doesn't really matter?

Reply:
Yeah we don't use aSPET but we already release ML51L and that one does. I think we just gotta wait til EJ57D goes out and remove ML51L.